html/semantics/scripting-1/the-script-element/moving-between-documents/tools/generate.py

```diff
@@ -19,11 +19,16 @@
 for timing in ["before-prepare", "after-prepare", "move-back"]:
   for destType in ["iframe", "createHTMLDocument"]:
     for result in ["fetch-error", "parse-error", "success"]:
-      for inlineOrExternal in ["inline", "external"]:
+      for inlineOrExternal in ["inline", "external", "empty-src"]:
         for type in ["classic", "module"]:
           # No fetch error for inline scripts.
           if result == "fetch-error" and inlineOrExternal == "inline":
             continue
+          if inlineOrExternal == "empty-src":
```

Comment:
It'd be good to comment why these combinations are skipped.

Reply:
Done